packages/client/package.json

```diff
@@ -183,7 +183,7 @@
     "@prisma/debug": "workspace:*",
     "@prisma/driver-adapter-utils": "workspace:*",
     "@prisma/engines": "workspace:*",
-    "@prisma/engines-version": "5.17.0-8.5954db1fa6f2b12aeb1d5cf503b870c3bb6f8387",
+    "@prisma/engines-version": "5.17.0-16.integration-dmmf-indexes-d964e5cfdd8811d32218d9bfa6c83522e0a1ddde",
```

Comment:
Switch to normal (non integration) version after merging the engines PR but before merging this PR